lib/libzfs/libzfs_pool.c

```diff
@@ -4169,13 +4168,59 @@ zpool_get_errlog(zpool_handle_t *zhp, nvlist_t **nverrlistp)
 	/*
 	 * Fill in the nverrlistp with nvlist's of dataset and object numbers.
 	 */
+	nvlist_t *nv;
+	uint64_t *block_ids;
+	int64_t *indrt_levels;
+	unsigned int error_count = 0;
 	for (i = 0; i < count; i++) {
-		nvlist_t *nv;
 
 		/* ignoring zb_blkid and zb_level for now */
 		if (i > 0 && zb[i-1].zb_objset == zb[i].zb_objset &&
-		    zb[i-1].zb_object == zb[i].zb_object)
+		    zb[i-1].zb_object == zb[i].zb_object) {
+			block_ids[error_count] = zb[i].zb_blkid;
+			indrt_levels[error_count] = zb[i].zb_level;
+			error_count++;
+			if (i == count - 1) {
+				if (nvlist_add_int64_array(nv, ZPOOL_ERR_LEVEL,
+				    indrt_levels, error_count) != 0) {
+					nvlist_free(nv);
+					goto nomem;
+				}
+				if (nvlist_add_uint64_array(nv,
+				    ZPOOL_ERR_BLOCKID, block_ids, error_count)
```

Comment:
Put the new line after the `!=`

Reply:
new line after != was exceeding 80 characters limit. So added new line after previous ,.